domenetjenester/beregningsgrunnlag/src/main/java/no/nav/foreldrepenger/domene/mappers/input/MapIAYTilKalkulusInput.java

```diff
@@ -99,10 +100,19 @@ public static InntektArbeidYtelseGrunnlagDto mapIAY(InntektArbeidYtelseGrunnlag
 
     private static ArbeidsforholdInformasjonDto mapTilArbeidsforholdInformasjonDto(ArbeidsforholdInformasjon arbeidsforholdInformasjon) {
         List<ArbeidsforholdOverstyringDto> resultat = arbeidsforholdInformasjon.getOverstyringer().stream()
-            .map(arbeidsforholdOverstyring -> new ArbeidsforholdOverstyringDto(mapTilAktør(arbeidsforholdOverstyring.getArbeidsgiver()),
-                arbeidsforholdOverstyring.getArbeidsforholdRef().gjelderForSpesifiktArbeidsforhold() ? new InternArbeidsforholdRefDto(arbeidsforholdOverstyring.getArbeidsforholdRef().getReferanse())
-                    : null,
-                KodeverkTilKalkulusMapper.mapArbeidsforholdHandling(arbeidsforholdOverstyring.getHandling())))
+            .map(arbeidsforholdOverstyring -> {
+                List<Periode> perioder = arbeidsforholdOverstyring.getArbeidsforholdOverstyrtePerioder() == null
```

Comment:
Ikke så glad i Optional.ofNullable(...).orElseGet(List::of).stream() ?

Reply:
Godt poeng, kan endre til det.